README.md

```diff
@@ -1,49 +1,86 @@
 # JSON Schema usage and problems
 
-Public forum to collect real world schemas and to discuss JSON Schema specification problems
-
 [![Build Status](https://travis-ci.org/epoberezkin/json-schema-usage-problems.svg?branch=master)](https://travis-ci.org/epoberezkin/json-schema-usage-problems)
 
+This repo is a public forum for collecting real-world schemas and discussing
+problems with the JSON Schema specification.
 
 ## Problems
 
-For JSON Schema authors and users, particularly those who are just starting to use JSON Schema, the specification is quite confusing. Validators, while complying with the spec, often produce the results that surprise users - ignored unknown keywords, type is not enforced, properties are optional, etc.
-
-For validator maintainers, the current draft-07 of the JSON Schema specification is already quite difficult to implement (See [Test of compliance of JS validators with the specification][1]). The upcoming draft-08 includes additional features that further increase implementation complexity.
-
-For tool maintainers, JSON Schema presents further challenges that have been partially addressed in OpenAPI version of the specification, but having two different versions of JSON Schema also creates additional problems for the users.
-
+At a high level, JSON Schema has three big problems:
+
+* **Bad default behavior.** For JSON Schema authors and users, particularly
+  those new to JSON Schema, the spec is very confusing and surprising. Some
+  particularly bad defaults are:
+
+  1. Keywords that are specific to particular data types, like `items` or
+    `properties`, are *ignored* if the inputted data isn't of the relevant data
+    type.
+  2. The `properties` keyword is actually about optional properties. To make a
+     property be required, you have to use the `required` keyword in combination
+     with `properties`. This also means you must give the name of the property
+     twice, and forgetting or mistyping a property name at either location will
+     lead to surprising results.
+  3. If you're using a JSON object as a dictionary (e.g. where the keys are IDs,
+    and the values are user objects), you can only validate that using
+    `additionalProperties`, and you must be careful to not use `properties` or
+    `patternProperties`, otherwise `additionalProperties` will sometimes be
+    ignored.
+  4. And much more.
+
+* **Difficulty of implementation.** The current draft of JSON Schema, known as
+  draft-07, is very difficult to implement. Through testing, we have determined
+  that [no two implementations of JSON Schema seem to resolve `$ref` in the same
+  way][1]. In draft-07, there are *34* keywords to support, and draft-08 will
+  add even more.
+
+* **Differing specs.** Because JSON Schema is complex while also failing to
```

Comment:
This argument works against the new `new-json-schema` (or whatever the working title is).

![standards](https://user-images.githubusercontent.com/973543/54110768-24bfec80-43da-11e9-8f1d-3ecf300f24a3.png)

I would remove it.